src/Entity/Mooc/AttachmentFile.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace AppBundle\Entity\Mooc;
+
+use AppBundle\EntityListener\Interfaces\EntityFileInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="mooc_attachment_file")
+ * @ORM\EntityListeners({"AppBundle\EntityListener\FileListener"})
+ */
+class AttachmentFile implements EntityFileInterface
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     *
+     * @Assert\NotBlank
+     */
+    private $title;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     */
+    private $path;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     */
+    private $extension;
+
+    /**
+     * @var UploadedFile|null
+     *
+     * @Assert\File
```

Comment:
what about set `maxSize`?

Reply:
![image](https://user-images.githubusercontent.com/1099956/41151961-9cb3f282-6b12-11e8-882e-9e2b3adbdbe8.png)

Reply:
![image](https://user-images.githubusercontent.com/1099956/41152508-1274e5ac-6b14-11e8-86ba-6b36743bcbc0.png)

Reply:
Julien said it's not needed